crates/ruma-events/src/space/child.rs

```diff
@@ -0,0 +1,65 @@
+//! Types for the *m.space.child* event.
+
+use ruma_events_macros::EventContent;
+use ruma_identifiers::ServerNameBox;
+use serde::{Deserialize, Serialize};
+
+use crate::StateEvent;
+
+/// The admins of a space can advertise rooms and subspaces for their space by setting
+/// `m.space.child` state events. The `state_key` is the ID of a child room or space, and the
+/// content must contain a `via` key which gives a list of candidate servers that can be used to
+/// join the room
```

Comment:
It would be nice if there was a short first paragraph before this big one, like in commit messages:

```plain
Short description that will be shown in the module docs.

Additional details that will only be shown when viewing the docs for this type.
```

Also there should be a period at the end.